source/github-helpers/index.ts

```diff
@@ -24,12 +24,13 @@ Example tag content on private repositories https://github.com/private/private/c
 export const getCurrentBranch = (): string | undefined => {
 	// .last needed for #2799
 	const feedLink = select.last('link[type="application/atom+xml"]');
```

Comment:
Feels like this shouldn't be missing. I hope it's temporary

Reply:
I wish, but it was removed from so many pages that I think its gone 😢

Reply:
I’m gonna ping some people on Twitter, hopefully someone answers

Reply:
Yes

Reply:
As an update, the feed is there, but only on iscommits